sql_metadb/derived_tables/po_notes.sql

```diff
@@ -0,0 +1,33 @@
+--metadb:table po_notes
+--Creates a derived table to show all notes attached to purchase orders, including general notes, renewal notes, and notes when purchase order is closed. 
+
+DROP TABLE IF EXISTS po_notes;
+CREATE TABLE po_notes AS
+SELECT
+po.id AS po_id,
+(po.jsonb -> 'metadata' ->> 'createdDate')::timestamptz AS created_date,
+po.jsonb ->> 'poNumber' AS po_number,
+po.jsonb ->> 'workflowStatus' AS po_workflow_status,
+po_notes.jsonb #>> '{}' AS po_note,
+po_notes.ORDINALITY AS po_note_ordinality,
```

Comment:
Please use all lowercase for identifiers.  See: https://github.com/folio-org/folio-analytics/blob/main/CONTRIBUTING.md#8-naming-things